docs/docs/concepts/eventstore/zitadel.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Implementation in ZITADEL
+---
+
+This documentation gives you an insight into the structure of the ZITADEL database.
+The goal is to give you a rough overview, so you know where which data is stored and which database schemas and tables are used.
+
+## Event
+
+The single source of truth of ZITADEL are the events that are stored in the eventstore. 
+From these events all different kind of resources e.g Users, Projects, Applications, etc. can be computed.
+
+An event has the following data:
+
+| Attribute | Description | Example |
+| --- | --- | --- |
+| id | unique identifier of the event, this is generated by the database  | b6402a60-e4655-4cc0-904b-f5c6760c4406 |
+| aggregate_type | the type of the aggregate, an aggregate can be compared to a resource or an object. One aggregate contains multiple event types | user |
+| aggregate_id | The unique identifier of an aggregate, this is generated by ZITADEL and is a sonyflake id | 168096909691353697 |
+| aggregate_version | The aggregate version shows in which version of the aggregate the event was created. This is needed to be able to compute correct objects | v1 |
+| event_type | The type of the event | user.human.added |
+| event_sequence | The event sequence is a sequence number that is incremented by one for each event on the instance. For technical reasons, a number can be omitted in some cases. This is needed so that the sequence of the events can be ensured. | 1234 | 
+| previous_aggregate_sequence| This number is the sequence of the event last created on this specific aggregate. E.g. Last user with specific aggregate_id | 1233|
+| previous_aggregate_type_sequence | This number is the sequence of the event last created on this aggregate. E.g Last User | 1230 |
```

Comment:
What are they used for? Sanity checks?

Reply:
they are used for the projections. They compare if the current sequence of the projection matches this sequence